data/test/scenarios/wml_tests/UnitsWML/AbilitiesWML/drains/drains_no_value.cfg

```diff
@@ -0,0 +1,40 @@
+#textdomain wesnoth-test
+
+#####
+# API(s) being tested: [drains]value=
+##
+# Actions:
+# Give the leaders a drains ability missing the value attribute.
+# Have the side 1 leader attack the side 2 leader with both its weapons
+##
+# Expected end state:
+# The both leaders have 2 weapons each of which now drains 50 hp (50% of 100 damage)
+# The leader of side 1 heals 50 hp total since it has full hp when its first strike is made
+# The leader of side 2 heals 100 hp total since it was damaged when its first strike was made
+#####
+{COMMON_KEEP_A_B_UNIT_TEST "drains_no_value" (
+    [event]
+        name = start
+
+        [modify_unit]
+            [filter]
+            [/filter]
+            [effect]
+                apply_to = new_ability
+                [abilities]
+                    [drains]
+                        id = "test-drains"
+                        name = _"test-drains"
+                        affect_self = yes
```

Comment:
personally I like this style of only putting relevant attributes more than TEST_ABILITY

Reply:
It's a bunch of duplicate code, which is what macros are made to help deal with. TEST_ABILITY also makes sure all attributes are set with the intended value and gives a standard ID for the ability.